src/main/groovy/com/marklogic/gradle/task/roxy/scaffoldTemplates/SampleTestFiles.groovy

```diff
@@ -0,0 +1,158 @@
+package com.marklogic.gradle.task.roxy
+
+class SampleTestFiles {
+
+    static String getSampleTestsXqy() {
+        return """
+xquery version '1.0-ml';
+
+import module namespace c = 'http://marklogic.com/roxy/test-config' at '/test/test-config.xqy';
+import module namespace test = 'http://marklogic.com/roxy/test-helper' at '/test/test-helper.xqy';
+import module namespace sec= 'http://marklogic.com/xdmp/security' at  '/MarkLogic/security.xqy';
+import module namespace cfg = 'http://thieme.com/synone/config' at '/app/config/config.xqy';
+
+
+declare option xdmp:mapping 'false';
+
+(: using global variable from app/config/config.xqy :)
+
+declare variable \\\$admin-user as xs:string := \\\$cfg:SEARCH-USER;
+declare variable \\\$admin-password as xs:string := \\\$cfg:SEARCH-PWD;
+
+
+xdmp:log('\$testName Tests BEGINNING.....')
+,
+(: using trace API - you can turn activate and deactivate this in the admin console :)
+xdmp:trace('ps-\$testName','Running in database ' || xdmp:database-name(xdmp:database()))
+,
+xdmp:trace('ps-\$testName','admin user =' || \\\$admin-user )
+
+,
+
+(: ***************************************************************
+
+Working with Roxy Unit Testing 
```

Comment:
I think this is content that should be on the Roxy website (or wherever the docs are / will be for Roxy-Test), but not in a generated test. I think a generated test should just show one simple assertion (that successfully passes, I don't believe the ones below will pass). The end user can be expected to remove these and consult the Roxy docs if assistance is needed. The goal is just to stub this out, not to explain how to use Roxy-Test.